### ISSUE TITLE
UX improvement on « Regenerate specific CPCSS » workflow

### ISSUE THREAD

**Issue by GeekPress** [wp-media/wp-rocket#2790]
**Describe the bug**
1. During the CPCSS generation, the loader isn't always present:
 
![Screenshot-1](https://p195.p4.n0.cdn.getcloudapp.com/items/wbuWKjjK/Screen%20Recording%202020-06-19%20at%2011.34%20AM.gif)

It can be confused as we could think we had a bug.

Can we have the loader during all the process?

2. The new « Revert back to the default CPCSS » button doesn't have the good style.

It should be a red link like that:

![Screenshot-2](https://p195.p4.n0.cdn.getcloudapp.com/items/o0uvmrlN/Capture%20d%E2%80%99e%CC%81cran%202020-06-19%20a%CC%80%2011.17.54.png)

To make this style, the CSS classes has to be applied: `components-button is-link is-destructive`

3. The confirmation message for reverting to the default CPCSS is quite confused: **Critical CSS file deleted successfully.**

Suggestions:
- **Specific Critical CSS file deleted successfully.**
- **Revert back to default CPCSS successfull**

**Backlog Grooming (for WP Media dev team use only)**
- [ ] Reproduce the problem
- [ ] Identify the root cause
- [ ] Scope a solution
- [ ] Estimate the effort

**Comment by crystinutzaa** [comment]
@GeekPress 
1. For the first issue: `During the CPCSS generation, the loader isn't always present` can you please send me steps to recreate it?

I don't think why this might happen. Only when CPCSS is generated it should remove the loader and display the other view (with delete link)

2. This is a an easy change - simply change the classes on  `Revert Back`

3. As for the confirmation message it's also an easy change.

**Comment by Tabrisrp** [comment]
@GeekPress See first question from Cristina above

**Comment by GeekPress** [comment]
@crystinutzaa For the steps to reproduce the issue, you can follow what I did in the demo.

I've just tried again on smashcoding website and I confirm the actual behavior.

@Tabrisrp Thanks for the follow up!

**Comment by Davidacu** [comment]
I would like to add the following:
It seems that for point 1.
>During the CPCSS generation, the loader isn't always present:

For what I noticed, when this happens, the specific CPCSS is never generated
notice is not always reproducible, in the video bellow it happened 2/4 times.
https://screencast-o-matic.com/watch/cYjOYOmgPM

and I believe that it's more likely to occur if **separate cache files for mobile devices** box disabled/unchecked.
when it's enabled I did not run into this issue.
